pkg/provisioners/provisioner_util_test.go

```diff
@@ -7,15 +7,14 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
```

Comment:
It would be great if you could fix the import ordering in your editor :)

imports from same pkg are usually in a separate group

Reply:
bloody Visual Studio Code. 

what are you using ? 

and any template to import ?